include/aws/common/linked_list.h

```diff
@@ -25,11 +25,23 @@ struct aws_linked_list_node {
     struct aws_linked_list_node *prev;
 };
 
+/**
+ * Checks that a node's next and prev pointers are NULL.
+ */
+AWS_STATIC_IMPL bool aws_linked_list_node_is_wiped(struct aws_linked_list_node *node) {
```

Comment:
Nuke this function in favor if `AWS_IS_ZEROED`

Reply:
Done